### PR TITLE
open dlpack in windows

### PR DESCRIPTION
**Pull request by yajiedesign**

### CODE REVIEW

**Comment by cjolivier01** [comment]
try triggering another build

File: CMakeLists.txt [hunk 1]
```diff
@@ -209,11 +213,15 @@ elseif(UNIX)
 endif()
 
 if(USE_LAPACK)
-  add_definitions(-DMXNET_USE_LAPACK=1)
-  list(APPEND mxnet_LINKER_LIBS lapack)
+  if(BLAS STREQUAL "Open" OR BLAS STREQUAL "open")
+    add_definitions(-DMXNET_USE_LAPACK=1)
```
Comment (cjolivier01) [inline CMakeLists.txt]: DO unit tests pass with this?  test_laop() failed on mine with LAPACK and open blas.

File: CMakeLists.txt [hunk 2]
```diff
@@ -209,11 +213,15 @@ elseif(UNIX)
...
+  elseif()
+    find_package(LAPACK REQUIRED)
```
Comment (cjolivier01) [inline CMakeLists.txt]: Does LAPACK require CBLAS (atlas) for CPU?

Comment (yajiedesign) [reply CMakeLists.txt]: ATLAS should be included a few LAPACK operations.But I don't know it is enough.

**Comment by yajiedesign** [comment]
@piiswrong

**Comment by yajiedesign** [comment]
@piiswrong

**Comment by piiswrong** [comment]
closing due to inactive

**Pull request closed without merging**